DylibBinder/DBInnerTypes.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using SwiftReflector;
+using SwiftReflector.Inventory;
+
+namespace DylibBinder {
+	internal class DBInnerTypes {
+		public DBInnerTypes (SwiftClassName swiftClassName)
+		{
+			var name = swiftClassName.ToString ();
+			if (DBTypeDeclarations.InnerXDictionary.ContainsKey (name)) {
```

Comment:
Please read about https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue?view=net-5.0 and remove the if.